torchdata/datapipes/iter/load/huggingface.py

```diff
@@ -36,10 +36,17 @@ class HuggingFaceHubReaderIterDataPipe(IterDataPipe[Tuple[str, StreamWrapper]]):
         streaming: Stream dataset instead of downloading it one go
         data_files: Optional dict to set custom train/test/validation split
     Example:
-        >>> from torchdata.datapipes.iter import IterableWrapper, HuggingFaceHubReaderIterDataPipe
-        >>> huggingface_reader_dp = HuggingFaceHubReaderDataPipe("lhoestq/demo1", revision="main")
-        >>> elem = next(iter(huggingface_reader_dp))
-        >>> elem["package_name"]
+
+    .. testcode::
```

Comment:
Interestingly HuggingFace seems neither to appear in the old nor in the new docs.

Reply:
I am sorry that I don't fully understand the statement here

Reply:
At the time HuggingFace was not rendered in the docs.